claat/proto-renderer/tests_utils.go

```diff
@@ -14,3 +18,48 @@ func NewDummyProto(in string) *devrel_tutorial.StylizedText {
 		Text: in,
 	}
 }
+
+// Type for Render() handling output
+type RenderedTemplate struct {
+	txt io.Reader
+	err error
+}
+
+// Type for canonical i != o template tests
+type TestingBatch struct {
+	// in  interface{}
+	In  *RenderedTemplate
```

Comment:
Privatize

Reply:
Can't privatize else I cannot use this in other files:
`implicit assignment of unexported field`

Reply:
The only issue on this is the following:

Per Travis, Go 1.11.x linter doesn't like this style: 
`composite literal uses unkeyed fields`
@cassierecher